test/remote/sms_tests.js

```diff
@@ -10,7 +10,7 @@ const Client = require('../client')()
 const config = require('../../config').getProperties()
 const error = require('../../lib/error')
 
-describe('remote sms with signinCodes disabled', function() {
+describe('remote sms without the signinCodes feature included in the payload', function() {
```

Comment:
I made an equivalent change to `test/local/routes/sms.js` in #1904 at @shane-tomlinson's behest, but forgot to update the descriptions for the remote tests too.